madmom/features/beats.py

```diff
@@ -48,38 +49,47 @@ class RNNBeatProcessor(SequentialProcessor):
 
     """
 
-    def __init__(self, post_processor=average_predictions, **kwargs):
+    def __init__(self, post_processor=average_predictions, online=False,
+                 nn_files=None, **kwargs):
```

Comment:
- Is there any use case for the `nn_files` parameter?
- Does it make sense to have the `nn_files` parameter if there is no `frame_sizes` parameter? As I see it, only neural networks trained on the given frame sizes (either for online or offline mode) would work.